src/utils.rs

```diff
@@ -185,21 +193,23 @@ mod tests {
             fs::remove_file(wallet_path).unwrap();
         }
     }
-    fn create_wallet(wallet_path: &Path) {
+    fn create_wallet(wallet_path: &Path, content: Option<&str>) {
```

Comment:
Perhaps a doc comment here would be helpful. Not obvious what `content` is expected from the function sig.

Reply:
addressed: https://github.com/FuelLabs/forc-wallet/pull/220/commits/0306d8382bb3adfa32acbcea368385601e23b00b